File: MyHomeSPA/src/app/shoppingPlanners/models/shoppingCard.ts
import { Purchase } from './purchase';
import { TypeOfPurchase } from './typeOfPurchase';
import { ShoppingCategory } from './shoppingCategory';

export class ShoppingCard {

    public purchase: Purchase;

    public typeOfPurchase: TypeOfPurchase;

    public shoppingCategory: ShoppingCategory;

    constructor() {

        this.purchase = new Purchase();

        this.typeOfPurchase = new TypeOfPurchase();

        this.shoppingCategory = new ShoppingCategory();
    }
}

File: ShoppingPlannerService/ShoppingPlannerService.WebApi/Models/ShoppingCard.cs
using Common.Entity.ShoppingPlannerService;

namespace ShoppingPlannerService.WebApi.Models
{
    public class ShoppingCard
    {
        public Purchase Purchase { get; set; }

        public TypeOfPurchase TypeOfPurchase { get; set; }

        public ShoppingCategory ShoppingCategory { get; set; }
    }
}
